src/services/RetroGenerationService.ts

```diff
@@ -86,8 +86,16 @@ class RetroGenerationService {
                     detailsTemplate.certificateNumber.value = testType.certificateNumber;
                     detailsTemplate.expiryDate.value = testType.testExpiryDate ? moment(testType.testExpiryDate).tz(TimeZone.LONDON).format("DD/MM/YYYY") : "";
                     detailsTemplate.preparerId.value = testResult.preparerId;
-                    detailsTemplate.failureAdvisoryItemsQAIComments.value = defects + reasonForAbandoning + additionalCommentsAbandon + "Additional test type notes: " +
-                                                                            additionalTestTypeNotes + ";\r\n" + (testType.additionalNotesRecorded ? (testType.additionalNotesRecorded + ";") : "");
+                    detailsTemplate.failureAdvisoryItemsQAIComments.value = defects
+                                                                            + reasonForAbandoning
+                                                                            + additionalCommentsAbandon
+                                                                            + (this.isPassingLECTestType(testType) ? (
```

Comment:
can we put this in the same way as `reasonForAbandoning` and `additionalCommentsAbandon`. Check and format string separately and then just map the value over here.